.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -3,7 +3,8 @@ Release checklist:
 Tagging:
  - [ ] Write release notes in NEWS. Get them reviewed and merged
      - [ ] If doing a branched release, also include a PR to merge the NEWS changes into master
- - [ ] Ensure your local copy is up to date with master and your working directory is clean
+ - [ ] Ensure your local copy is up to date with master branch (`git@github.com:coreos/ignition.git`)
```

Comment:
I think the word `upstream` from https://github.com/coreos/ignition/pull/1120#discussion_r513908112 is useful to include, for clarity.